src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ShippingMethodRepository.php

```diff
@@ -69,7 +70,8 @@ protected function createEnabledForChannelQueryBuilder(ChannelInterface $channel
     {
         $queryBuilder = $this
             ->createQueryBuilder('o')
-            ->where('o.enabled = true')
+            ->andWhere('o.enabled = true')
+            ->andWhere('o.archivedAt IS NULL')
```

Comment:
Usually I'd be for it all the way, but in this case I doubt it'd be worth using expressions for something so simple.